src/QsCompiler/Transformations/QsharpCodeOutput.cs

```diff
@@ -528,6 +528,49 @@ private static string ReplaceInterpolatedArgs(string text, Func<int, string> rep
                 return InterpolationArg.Replace(text, ReplaceMatch);
             }
 
+            private static string QsSymbolToString(QsSymbol symbol)
+            {
+                if (symbol.Symbol is QsSymbolKind<QsSymbol>.Symbol sym)
+                {
+                    return sym.Item;
+                }
+                else if (symbol.Symbol is QsSymbolKind<QsSymbol>.QualifiedSymbol qSym)
+                {
+                    return $"{qSym.Item1}.{qSym.Item2}";
+                }
+                else if (symbol.Symbol is QsSymbolKind<QsSymbol>.SymbolTuple tup)
+                {
+                    if (tup.Item.Length == 0)
+                    {
+                        return "()";
+                    }
+                    else if (tup.Item.Length == 1)
+                    {
+                        return QsSymbolToString(tup.Item.First());
+                    }
+                    else
+                    {
+                        return $"({string.Join(", ", tup.Item.Select(QsSymbolToString))})";
+                    }
+                }
+                else if (symbol.Symbol.IsOmittedSymbols)
+                {
+                    return "__omittedSymbol__";
```

Comment:
Omitted symbols are generally three dots, so you can replace that with "..."

Reply:
updated, thanks